packages/react-router/src/router.ts

```diff
@@ -825,6 +825,10 @@ export class Router<
     return this.routesById as Record<string, AnyRoute>
   }
 
+  get looseRoutesByPath() {
+    return this.routesByPath as Record<string, AnyRoute>
+  }
```

Comment:
this becomes a public function exposed to users. I don't think we'd want that. Since there's only one usage, why not just inline it?

Reply:
Inlined